src/vm/eventpipebuffermanager.h

```diff
@@ -15,6 +15,33 @@
 
 class EventPipeBufferList;
 
+// This class is a TLS wrapper around a pointer to thread-specific EventPipeBufferList
+// The struct wrapper is present mainly because we need a way to free the EventPipeBufferList
+// when the thread that owns it dies. Placing this class as a TLS variable will call ~ThreadEventBufferList()
+// when the thread dies so we can free EventPipeBufferList in the destructor.  
+class ThreadEventBufferList
+{
+public:
```

Comment:
Nit: Since you have get&set methods, the field itself can be private. Only the methods need to be public.